pyseir/inference/infer_rt.py

```diff
@@ -242,7 +236,7 @@ def apply_gaussian_smoothing(self, timeseries_type, plot=False, smoothed_max_thr
         """
         timeseries_type = TimeseriesType(timeseries_type)
         dates, times, timeseries = self.get_timeseries(timeseries_type)
-        bind_threadlocal(timeseries_type=timeseries_type.value)
+        self.log = self.log.bind(timeseries_type=timeseries_type.value)
```

Comment:
Are these changes already in Alex's code? If so, I prefer we just merge his branch, rather than having the infer_rt.py changes across multiple PRs.

Reply:
These are different changes. I'll be happy to work through the merge conflicts if they show up.